custom-completions/mod.nu

```diff
@@ -0,0 +1,26 @@
+export use ani-cli/ani-cli-completions.nu *
+export use git/git-completions.nu *
```

Comment:
you have `git` twice, you can remove this one i think :+1:

Reply:
true! I deleted it in [60f0fd7](https://github.com/nushell/nu_scripts/pull/603/commits/60f0fd782ae41d9e1ab76d90c4844769f939b29a)

Reply:
thanks :pray: